<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KeyEditor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>KeyEngine.ico</normaloff>KeyEngine.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QTextEdit" name="CentralView"/>
  <widget class="QMenuBar" name="MenuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="NewProject"/>
    <addaction name="OpenProject"/>
    <addaction name="Save"/>
    <addaction name="SaveAll"/>
    <addaction name="separator"/>
    <addaction name="OpenVS"/>
    <addaction name="Build"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="MenuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="Undo"/>
    <addaction name="Redo"/>
    <addaction name="separator"/>
    <addaction name="Cut"/>
    <addaction name="Copy"/>
    <addaction name="Paste"/>
    <addaction name="separator"/>
    <addaction name="Duplicate"/>
    <addaction name="Delete"/>
    <addaction name="separator"/>
    <addaction name="Find"/>
    <addaction name="SelectAll"/>
    <addaction name="separator"/>
    <addaction name="EditorPreferences"/>
    <addaction name="ProjectSettings"/>
   </widget>
   <widget class="QMenu" name="MenuGameObject">
    <property name="title">
     <string>GameObject</string>
    </property>
    <addaction name="CreateGameObject"/>
   </widget>
   <widget class="QMenu" name="MenuComponent">
    <property name="title">
     <string>Component</string>
    </property>
    <addaction name="AddComponent"/>
   </widget>
   <widget class="QMenu" name="MenuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="OpenDocumentation"/>
    <addaction name="separator"/>
    <addaction name="OpenAbout"/>
   </widget>
   <widget class="QMenu" name="MenuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="OpenInspector"/>
    <addaction name="OpenOutliner"/>
    <addaction name="OpenContent"/>
    <addaction name="OpenConsole"/>
   </widget>
   <widget class="QMenu" name="MenuTypes">
    <property name="title">
     <string>Types</string>
    </property>
   </widget>
   <addaction name="MenuFile"/>
   <addaction name="MenuEdit"/>
   <addaction name="MenuGameObject"/>
   <addaction name="MenuComponent"/>
   <addaction name="MenuWindow"/>
   <addaction name="MenuHelp"/>
   <addaction name="MenuTypes"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <widget class="QToolBar" name="MainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Translate"/>
   <addaction name="Rotate"/>
   <addaction name="Scale"/>
   <addaction name="separator"/>
   <addaction name="Compile"/>
   <addaction name="Build"/>
  </widget>
  <widget class="QDockWidget" name="Outliner">
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>93</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Outliner</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="Inspector">
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>38</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="ContentBrowser">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>200</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Content Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
  </widget>
  <action name="NewProject">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/new_project.png</normaloff>:/icons/new_project.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="OpenProject">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/open_project.png</normaloff>:/icons/open_project.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="Save">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="Build">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/build.png</normaloff>:/icons/build.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="Exit">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="Undo">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="Redo">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="Cut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="Copy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="Paste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="Duplicate">
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="Delete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="Find">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="EditorPreferences">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/editor_preferences.png</normaloff>:/icons/editor_preferences.png</iconset>
   </property>
   <property name="text">
    <string>Editor Preferences</string>
   </property>
  </action>
  <action name="ProjectSettings">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/project_settings.png</normaloff>:/icons/project_settings.png</iconset>
   </property>
   <property name="text">
    <string>Project Settings</string>
   </property>
  </action>
  <action name="CreateGameObject">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/outliner/gameobject.png</normaloff>:/icons/outliner/gameobject.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+N</string>
   </property>
  </action>
  <action name="AddComponent">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="OpenDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="OpenAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="SelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="OpenInspector">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/inspector.png</normaloff>:/icons/inspector.png</iconset>
   </property>
   <property name="text">
    <string>Inspector</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="OpenOutliner">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/scene_outliner.png</normaloff>:/icons/scene_outliner.png</iconset>
   </property>
   <property name="text">
    <string>Outliner</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="OpenContent">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/content.png</normaloff>:/icons/content.png</iconset>
   </property>
   <property name="text">
    <string>Content</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="OpenConsole">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/console.png</normaloff>:/icons/console.png</iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="SaveAll">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/save_all.png</normaloff>:/icons/save_all.png</iconset>
   </property>
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="Translate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/translate.png</normaloff>:/icons/translate.png</iconset>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
   <property name="toolTip">
    <string>Translate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="Rotate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/rotate.png</normaloff>:/icons/rotate.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="toolTip">
    <string>Rotate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="Scale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/scale.png</normaloff>:/icons/scale.png</iconset>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
   <property name="toolTip">
    <string>Scale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="Compile">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/compile.png</normaloff>:/icons/compile.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="toolTip">
    <string>Compile</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="OpenVS">
   <property name="icon">
    <iconset resource="../Resources/Icons/icons.qrc">
     <normaloff>:/icons/visual_studio.png</normaloff>:/icons/visual_studio.png</iconset>
   </property>
   <property name="text">
    <string>OpenVS</string>
   </property>
   <property name="toolTip">
    <string>OpenVS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/Icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>OpenOutliner</sender>
   <signal>triggered()</signal>
   <receiver>Outliner</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenInspector</sender>
   <signal>triggered()</signal>
   <receiver>Inspector</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>895</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenContent</sender>
   <signal>triggered()</signal>
   <receiver>ContentBrowser</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>640</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
